docker-compose.yml

```diff
@@ -0,0 +1,24 @@
+version: "3.6"
+
+services:
+  api:
+    build: .
+    ports:
+      - 3000:3000
+    depends_on:
+      - mongodb
+    environment:
+      - PORT=3000
```

Comment:
There are quite a few environment variables the container will need. Can you edit this so it reads in from the `.env` file? You can see all the variables in the `.env.sample` file.

See here for an example of pulling in a value from the `.env`: https://github.com/CodingGarden/entropychat.app/blob/master/server/docker-compose.yml#L28

Reply:
what about using
`env_file:
      - .env
` 
just for the sake of verbosity